routes/web.php

```diff
@@ -41,3 +42,5 @@
 Route::post('/product/create', [ProductController::class, 'store'])->name('storeProduct');
 
 
+Route::get('/add/cart/{id}', [CartController::class, 'show'])->name('cart');
+Route::get('/my/cart', [CartController::class, 'create'])->name('myCart');
```

Comment:
This is not correct routing na... you can use cart/show.. what is my/cart? Does it sense to your ear?

Reply:
Corrected Sir. Noted!